packages/espree/package.json

```diff
@@ -56,13 +61,16 @@
     "acorn"
   ],
   "scripts": {
-    "build": "rollup -c rollup.config.js",
+    "lint:types": "attw --pack",
+    "tsc": "tsc && tsc -p tsconfig-cjs.json",
+    "build": "rollup -c rollup.config.js && npm run tsc && rm -Rf dist/lib",
```

Comment:
It's better to use JavaScript code to delete the directory because `rm -Rf` doesn't work in PowerShell. You could for example create a script in `packages/espree/tools`. An example is in the `sync-docs` tool:

https://github.com/eslint/js/blob/6f4d32568c247978153105af2f2a59989abc55d7/packages/espree/tools/sync-docs.js#L26

Another option is using `node -e "fs.rm('dist/lib')"`. We are doing something similar in the rewrite package to copy a directory:

https://github.com/eslint/rewrite/blob/0f16298e06e0380f259611c7786a195a4627ee34/packages/core/package.json#L20